lib/CompositeController.js

```diff
@@ -141,13 +141,8 @@ module.exports = class CompositeController extends BaseController {
       };
     }
 
-    // TODO: https://github.com/razee-io/razeedeploy-core/issues/189
     let impersonateUser = objectPath.get(this.data, 'object.spec.clusterAuth.impersonateUser', 'razeedeploy');
-    if (impersonateUser !== 'razeedeploy' && this.namespace === 'razeedeploy') {
-      krm.addHeader('Impersonate-User', impersonateUser);
-    } else {
-      impersonateUser = 'razeedeploy';
-    }
+    krm.addHeader('Impersonate-User', impersonateUser);
```

Comment:
if impersonateUser is razeedeploy, we **shouldn't** add the header.

Reply:
addressed.